docs/build-on-polkadot/parachains/asset-hub/register-a-local-asset.md

```diff
@@ -0,0 +1,88 @@
+
+4. Once you have confirmed that the asset ID is unique, click on the **Create** button on the top right corner of the page
+   
+      ![Create a new asset](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-3.webp)
+
+5. Fill in the required fields in the **Create Asset** form and click on the **Next** button
+   
+    - `creator account` - the account to be used for creating this asset and setting up the initial metadata
+    - `asset name` - the descriptive name of the asset you are registering
+    - `asset symbol` - the symbol that will be used to represent the asset
+    - `asset decimals` - the number of decimal places for this token, with a maximum of 20 allowed through the user interface
+    - `minimum balance` - the minimum balance for the asset. This is specified in the units and decimals as requested
+    - `asset ID` - the selected id for the asset. This should not match an already-existing asset id
+ 
+    ![Create Asset Form](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-4.webp)
+
+6. Select the `admin account`, `issuer-account` and `freezer-account`, and then click on the **Create** button
+
+    - `admin account` - the account designated for continuous administration of the token      
+    - `issuer account` - the account that will be used for issuing this token
+    - `freezer account` - the account that will be used for performing token freezing operations
+
+    ![Admin, Issuer, Freezer accounts](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-5.webp)
+
+8. Finally, click on the **Sign and Submit** button to complete the asset registration process.
+
+    ![Sign and Submit](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-6.webp)
+
+## Verify asset registration
+
+After completing these steps, the asset will be successfully registered. You can now view your asset listed on the [`Assets` section](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/assets){target=_blank} of the Polkadot.js Apps interface.
+      
+![Asset listed on Polkadot.js Apps](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-7.webp)
+
+!!! note
+    Take into consideration that the `Assets section` link will be different according to the network you are using. For the local environment, the link will be `ws://127.0.0.1:8000/#/assets`.
+
+In this way, you have successfully registered a local asset on the Asset Hub parachain.
+
+## Environment setup
```

Comment:
I think it's important to include a brief explanation somewhere about how to simulate this process since it involves real money in the real world. I initially moved it to the bottom, but perhaps I should create a separate file just for that purpose

However, the "registering a foreign asset" process would also require to have a test setup for that. I'm not sure if it makes sense to have separate test setup files.

Reply:
I think this just needs some tweaking and it will make more sense. It's good to show them how to run a test before going on to the live networks.

Reply:
Yes, I agree with that as well. I'm figuring out how to make it look better or make more sense.

I'm still unsure if the best approach would be to move this to a new file (considering that I'd have to do the same for the registration of foreign assets) or to add some extra introduction to this section to put the reader in context...

Any ideas? @0xLucca @dawnkelly09 @eshaben